.github/workflows/appsignals-e2e-test.yml

```diff
@@ -20,10 +20,10 @@ permissions:
 
 env:
   AWS_DEFAULT_REGION: us-east-1
-  TEST_ACCOUNT: ${{ secrets.APP_SIGNALS_E2E_TEST_ACCOUNT }}
+  TEST_ACCOUNT: ${{ secrets.APP_SIGNALS_E2E_TEST_ACCOUNT_ID }}
```

Comment:
Gave CWAgent team the same secret names as the CloudWatch [repo](https://github.com/aws/amazon-cloudwatch-agent), so they updated it with new secrets. I think this might be better though for consistency with the CloudWatch repo.